sjb/config/test_cases/test_pull_request_ovn_kubernetes_rhel.yml

```diff
@@ -0,0 +1,35 @@
+---
+parent: 'common/test_cases/ovn-kubernetes.yml'
+overrides:
+  provision:
+    os: "rhel"
+    stage: "ovn"
+    provider: "aws"
+extensions:
+  actions:
+  - type: "forward_parameters"
+    parameters:
+      - PULL_NUMBER
+      - PULL_BASE_SHA
+      - PULL_PULL_SHA
+  - type: "script"
+    title: "clone ovn-kubernetes and check out the correct refs"
+    script: |-
+      cd /go/src/github.com/openvswitch/ovn-kubernetes
+      git fetch origin
```

Comment:
You don't need to name your refs:

```sh
repo="https://github.com/openvswitch/ovn-kubernetes.git"
git init
git fetch "${repo}" "${PULL_BASE_REF}"
git checkout "${PULL_BASE_SHA:-'FETCH_HEAD'}"
git fetch "${repo}" "pull/${PULL_NUMBER}/head"
git merge "${PULL_PULL_SHA:-'FETCH_HEAD'}"
```

But if you want to handle batch jobs this gets more complicated...

Reply:
Just copied the stuff. I doubt if we need batch jobs.